RNTrueLayerPaymentsSDK/js/TrueLayerPaymentsSDKWrapper.ts

```diff
@@ -44,18 +46,21 @@ export abstract class TrueLayerPaymentsSDKWrapper {
    */
   static processPayment(
     paymentContext: PaymentContext,
-    preferences?: PaymentPreferences
+    preferences?: PaymentPreferences,
+    theme?: Theme
```

Comment:
The theme is part of configure, definitely not part of process payment, and process mandate.

Reply:
Updated to pass the theme via configure